release/version/version.go

```diff
@@ -34,23 +30,26 @@ func Parse(desc string) (Version, error) {
 	}
 
 	parts = strings.Split(num, ".")
-	if len(parts) != 3 {
-		return Version{}, fmt.Errorf("expected three parts in version number, got %s", num)
-	}
 
 	maj, err := strconv.Atoi(parts[0])
 	if err != nil {
 		return Version{}, fmt.Errorf("failed to parse major version %q", parts[0])
 	}
 
-	min, err := strconv.Atoi(parts[1])
-	if err != nil {
-		return Version{}, fmt.Errorf("failed to parse minor version %q", parts[1])
+	min := 0
```

Comment:
(nit) `min` is a builtin in go1.21. It's fine to use it (it works), but it's maybe confusing because editors will highlight it differently; I might use `minor` here instead.